index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" type="text/css" href="styles.css">
+
+
+</head>
+<body>
+    <nav class="navigation">
+       <img src="./logos/MyLogo.png" alt="navigation icon" class="navigation__logo">
+       <img src="./logos/menu.svg" alt="navigation icon" class="navigation__hamburger">
+    </nav>
+    
+<div class="bg">
+    <h1>I’m Mariane
+        Glad to see you!</h1>
+    <P>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</P>
+    <a href="">LET'S CONNECT</a>
+    <ul class="icons">
+        <li class="icons_list"><img src="./logos/Vector-4.png" alt="navigation icon" ></li>
+        <li class="icons_list"><img src="./logos/Vector-2.png" alt="navigation icon"></li>
+        <li class="icons_list"><img src="./logos/Vector-1.png" alt="navigation icon"></li>
+        <li class="icons_list"><img src="./logos/Vector-3.png" alt="navigation icon"></li>
+        <li class="icons_list"><img src="./logos/Vector.png" alt="navigation icon"></li>
+    </ul>
```

Comment:
- [ ] You should wrap the `<img>` tags within the `<li>` tags with `<a>` tag to make them clickable and help the user navigate to your corresponding social media profiles e.g.
```
<ul>
  <li>
    <a>
       <img src="./logos/Vector.png" alt="navigation icon">
    </a>
   </li>
  <li> ... </li>
</ul>
```